poshc2/client/command_handlers/ImplantHandler.py

```diff
@@ -477,6 +491,49 @@ def do_show_urls(user, command):
     clear()
 
 
+def do_show_hosted_files(user, command):
+    files = update_cache_urls()
+    filesformatted = "ID  URI  FilePath  ContentType  Base64  Active\n"
+    for i in files:
+        filesformatted += "%s  %s  %s  %s  %s %s \n" % (i[0], i[1], i[2], i[3], i[4], i[5])
+    print_good(filesformatted)
+    input("Press Enter to continue...")
+    clear()
+
+
+def do_add_hosted_file(user, command):
+    URI = input("URI Path: downloads/2020/application.docx ")
+    FilePath = input("File Path: .e.g. /tmp/application.docx ")
+    ContentType = input("Content Type: .e.g. text/html ")
+    Base64 = input("Base64 Encode File (Yes/No): ")
+    insert_hosted_file(URI, FilePath, ContentType, Base64, "Yes")
+    print_good("add-hosted-file %s -> %s (%s)\r\n" % (URI, FilePath, ContentType))
+    input("Press Enter to continue...")
+    clear()
+
+
+def do_del_hosted_file(user, command):
+    hosted_file_id = command.lower().replace("del-hosted-file ", "")
```

Comment:
We can do this in one with `hosted_file_id = command.lower().replace("del-hosted-file", "").strip()`.

We already know this is ok and it's not that *del-hosted-file* appears somewhere else in the command as we check that before calling `do_del_hosted_file`.